spec/syncCrudSpec.js

```diff
@@ -1,25 +1,47 @@
 const datasetId = 'specDataset';
+const datasetOneId = 'specDatasetOne';
+const datasetTwoId = 'specDatasetTwo';
 const testData = { test: 'text' };
 const updateData = { test: 'something else' };
 
 describe('Sync Create/Update/Delete', function() {
 
   beforeAll(function(done) {
+    localStorage.clear();
     $fh.cloud({
       path: '/datasets',
       data: {
         name: 'specDataset',
-        options: { syncFrequency: 1 }
+        options: { syncFrequency: 0.5 }
       }
     }, done, done.fail);
   });
 
   beforeEach(function() {
-    $fh.sync.init({ sync_frequency: 1, storage_strategy: 'dom' , crashed_count_wait: 1});
+    $fh.sync.init({ sync_frequency: 0.5, storage_strategy: 'dom' , crashed_count_wait: 1});
   });
 
   afterEach(function(done) {
-    $fh.sync.stopSync(datasetId, done, done.fail);
+    var datasets = [datasetId, datasetOneId, datasetTwoId];
+    var datasetsStopped = 0;
+
+    function datasetStopped() {
+      datasetsStopped++;
+      if (datasetsStopped === datasets.length) {
+        localStorage.clear();
+        return done();
+      }
+    }
+
+    datasets.forEach(function(dataset) {
+      $fh.sync.stopSync(dataset, function() {
+        datasetStopped();
+      }, function() {
+        console.warn('Problem stopping sync for dataset ' + dataset);
```

Comment:
@aidenkeating I think we can lower it to info and update it to say 'Its OK to ignore this, unless the dataset was actually used in your test'